### PR TITLE
New pages

### PR DESCRIPTION
**Pull request by Mussuranae**
3 new pages : contact, who are we and about pages.
Related to #1170

### CODE REVIEW

File: apps/catalog/marketplace/marketplace/src/app/pages/contact-page/contact-page.component.html [hunk 1]
```diff
@@ -0,0 +1,23 @@
+<header>
+  <h1>Contact</h1>
+</header>
+
+<div fxLayout="row" fxLayoutAlign="center">
```
Comment (Hugo-Hontans) [inline apps/catalog/marketplace/marketplace/src/app/pages/contact-page/contact-page.component.html]: Remove useless div

File: apps/catalog/marketplace/marketplace/src/app/pages/team-page/team-page.component.html [hunk 2]
```diff
@@ -0,0 +1,33 @@
+<header fxLayout="column" fxLayoutAlign="center center">
+  <h1>Who are we ?</h1>
+</header>
+
+<div  fxLayout="row" fxLayoutAlign="center">
```
Comment (Hugo-Hontans) [inline apps/catalog/marketplace/marketplace/src/app/pages/team-page/team-page.component.html]: same

File: apps/catalog/marketplace/marketplace/src/app/pages/team-page/team-page.component.html [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+  <mat-card>
+    <section>
```
Comment (Hugo-Hontans) [inline apps/catalog/marketplace/marketplace/src/app/pages/team-page/team-page.component.html]: article has more sens here

File: apps/catalog/marketplace/marketplace/src/app/pages/contact-page/contact-page.component.html [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+  <mat-card>
+    <section>
```
Comment (Hugo-Hontans) [inline apps/catalog/marketplace/marketplace/src/app/pages/contact-page/contact-page.component.html]: Use an article to keep the same logic than your `about-page`

File: apps/catalog/marketplace/marketplace/src/app/app-routing-module.ts [hunk 5]
```diff
@@ -66,6 +66,21 @@ export const routes: Routes = [
             path: 'catalog',
             children: [
               { path: '', redirectTo: 'home', pathMatch: 'full' },
+              {
+                path: 'about',
+                loadChildren: () =>
+                import('./pages/about-page/about-page.module').then(m => m.AboutPageModule)
```
Comment (Hugo-Hontans) [inline apps/catalog/marketplace/marketplace/src/app/app-routing-module.ts]: We never name a Module `Page`. Name it `AboutModule`.

File: apps/catalog/marketplace/marketplace/src/app/pages/about-page/about-page.component.html [hunk 6]
```diff
@@ -0,0 +1,47 @@
+<header>
+  <h1>About</h1>
+</header>
+
+<div fxLayout="row" fxLayoutAlign="center">
+<mat-card>
+  <article>
+    <h3>What is Bigger Boat?</h3>
+    <p>
+      Bigger Boat is the first marketplace offering package deals for TV and Film rights.
```
Comment (Hugo-Hontans) [inline apps/catalog/marketplace/marketplace/src/app/pages/about-page/about-page.component.html]: Ask Vincent if you have to rename it Archipel Media

File: apps/catalog/marketplace/marketplace/src/app/pages/about-page/about-page.component.html [hunk 7]
```diff
@@ -0,0 +1,47 @@
+<header>
+  <h1>About</h1>
+</header>
+
+<div fxLayout="row" fxLayoutAlign="center">
+<mat-card>
```
[the remark this answers is not in the document]
Comment (Mussuranae) [reply apps/catalog/marketplace/marketplace/src/app/pages/about-page/about-page.component.html]: Yes but only the design :/

File: apps/catalog/marketplace/marketplace/src/app/pages/contact-page/contact-page.component.html (hunk 1 above)
Comment (Mussuranae) [reply apps/catalog/marketplace/marketplace/src/app/pages/contact-page/contact-page.component.html]: It's not an useless div, I need it to align correctly the card in the page :o

File: apps/catalog/marketplace/marketplace/src/app/pages/team-page/team-page.component.html (hunk 2 above)
Comment (Mussuranae) [reply apps/catalog/marketplace/marketplace/src/app/pages/team-page/team-page.component.html]: Same here, I need it to use Flexbox and align the card correctly

Comment (Mussuranae) [reply apps/catalog/marketplace/marketplace/src/app/pages/team-page/team-page.component.html]: Same here, I need it to use Flexbox and align the card correctly

**Pull request merged**